### ISSUE TITLE
basic docs 

### ISSUE THREAD

**Issue by jstac** [lambdalisue/jupyter-vim-binding#5]
Really just came here to say thank you thank you thank you!

Just a couple of lines of basic documentation on the top page would be very helpful.  (The video is a bit hard to follow.)

**Comment by lambdalisue** [comment]
Well, what kind of basic documentation do you recommend to add?

**Comment by lambdalisue** [comment]
https://github.com/lambdalisue/jupyter-vim-binding#customize

I think I can close this issue now.
